src/components/TransactionItem/TransactionRGBPPDigestModal/TransactionRGBPPDigestTransfer.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from './styles.module.scss'
+import { parseBTCAddress } from '../../../utils/bitcoin'
+import { TransactionRGBPPDigestTransferAsset } from './TransactionRGBPPDigestTransferAsset'
+import { TransactionRecord } from '../../../services/ExplorerService'
+import AddressText from '../../AddressText'
+
+export const TransactionRGBPPDigestTransfer = ({ transfer }: { transfer: TransactionRecord }) => {
+  const address = parseBTCAddress(transfer.address)
+
+  let addressType = ''
+  if (address) {
+    addressType = address.isBech32 ? `Bech32 (${address.type})` : `Base58 (${address.type})`
```

Comment:
The condition is `encoding` is `bech32` or `not bech32` but the else branch treats `not bech32` as `is base58`, which is not proper. `encoding` will fallback to `bech58` when new `encoding` is introduced while this code is not updated.